tests/common.go

```diff
@@ -3591,6 +3662,24 @@ func DeleteSchedule(backupScheduleName string, clusterName string, orgID string,
 	return nil
 }
 
+// DeleteScheduleWithUID deletes backup schedule with the given backup schedule name and uid
+func DeleteScheduleWithUID(backupScheduleName string, backupScheduleUid string, orgID string, ctx context1.Context) error {
+	backupDriver := Inst().Backup
+	bkpScheduleDeleteRequest := &api.BackupScheduleDeleteRequest{
+		OrgId: orgID,
+		Name:  backupScheduleName,
+		// DeleteBackups indicates whether the cloud backup files need to
+		// be deleted or retained.
+		DeleteBackups: true,
+		Uid:           backupScheduleUid,
+	}
+	_, err := backupDriver.DeleteBackupSchedule(ctx, bkpScheduleDeleteRequest)
```

Comment:
If you want to delete the schedule as a user and then wait for its deletion as an admin, DeleteScheduleWithUID alone would not work. We would need both DeleteScheduleWithUID and DeleteScheduleWithUIDAndWait. But since we did not implement a DeleteScheduleAndWait, I did not add DeleteScheduleWithUIDAndWait.

Reply:
That's not a general use case to delete schedule from one user and wait from admin. The current DeleteSchedule has wait within, you can have the same implemented here.

Reply:
Implemented DeleteScheduleWithUIDAndWait. Thanks.